web/src/routes/(user)/photos/+page.svelte

```diff
@@ -26,11 +26,12 @@
 
   let { isViewing: showAssetViewer } = assetViewingStore;
   let handleEscapeKey = false;
-  const assetStore = new AssetStore({ isArchived: false, withStacked: true });
+  const assetStore = new AssetStore({ isArchived: false, withStacked: true, withPartners: true });
   const assetInteractionStore = createAssetInteractionStore();
   const { isMultiSelectState, selectedAssets } = assetInteractionStore;
 
   $: isAllFavorite = Array.from($selectedAssets).every((asset) => asset.isFavorite);
+  $: hasPartnerAssetsInSelection = Array.from($selectedAssets).some((asset) => asset.ownerId != data.user.id);
```

Comment:
Why not use `$selectedAssets.has(...)` instead of transforming it to an array first? The set method .has should do the same thing

Reply:
Isn't that only to check a specific key?

Reply:
My bad, sorry.